frontend/src/router/routeHandlers.ts

```diff
@@ -29,15 +30,27 @@ import type { RouteLocationNormalized } from 'vue-router';
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
+    let applicationAdmins;
+    let userRolesFetchResult;
     // Requires fetching applications the user administers.
     await asyncWrap(fetchApplications());
-    const userRolesFetchResult = await asyncWrap(
-        fetchUserRoleAssignments(selectedApplication.value?.application_id)
-    );
-    Object.assign(to.meta, { userRoleAssignments: userRolesFetchResult.data });
+    if(selectedApplication.value?.application_id === 1) {
```

Comment:
If I code this as **"1"** or **"2"**, you would not know what **"1"** or **"2"** are for if you just see it.

However, even if I knew, I would not use **"1"** or **"2"** in the logic, since they are meaningless and they are randomly assigned (a sequence id from database), they are not really reliable and controlled by us. In some cases or in different environments, your "**2**" maybe be "**5**" (although in our case, it is not the case), still it is dangerous to use. I would use some "code" (we don't have) or "application name" (at least not likely it will be changed, but you could put a comment for developer to be aware).

And I would only trust using application name with "1" = "FAM" (exceptional case), I would not trust "2" is "FOM_DEV". But, all with above saying, please use "FAM" not "1".

Reply:
thanks for the comment, makes sense.